magsnake.md

```diff
@@ -0,0 +1,131 @@
+---
+name: "@magsnake"
+project: "  Multi-purpose Smart Robot"
+---
+
+# Project Name
+
+  Multi-purpose Smart Robot
+
+## Summary
+
+  I am going to build a mini robot which can solve questions, make decisions, manage digital activities and serve as a virtual assistant.
+  
+  It is very useful since it can be used to solve my studies related problems and it can create a customized planner for me.
+  
+  I am very excited to build this project.
+
+  It can detect objects, search images, manage app time and their activities for other devices, serve as AI Assistant ad can tell climatic conditions.
+  
+  It can be used as a pass-key for my accounts. All this will save a lot of time and will be very useful.
+  
+  It could manage networks and I will make it to do a lot more things.
+
+## Plan
+
+  For the ML software, I have Google Cloud ML Resources with Tensorflow. This will help me using APIs to make the robot interactive.
+  I have Arduino IDE for arduino development.
+  
+  I will make it with the help of Raspberry Pi 4 and Arduino Mega.
+  
+  I will use their respective Orange Kit (A kit consisting of all necessary components used for making the projects) which will come with all other necessary materials such as (Displays, Adapters, Connectors, Wires, Resistnces, Buttons etc.)
+  
+  I will use the respective APIs for making it do the things such as Playing board Games, Solving Problems, Getting me make a day planner with Pomodoro.
```

Comment:
which respective APIs?

Reply:
I meant that for Different board games, I am gonnause different APIs.
Such as chess.com API for chess and Sudoku generator API for sudoku etc.
I will also use TensorFlow API Library for Object Detection ad Voice Recognition  etc.